src/PFire.Core/Models/GameModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PFire.Core.Models
+{
+    public class GameModel
+    {
+        public int GameID { get; set; }
```

Comment:
Like the user model, since we now know this is game, we can call these properties `Id, `Ip`, `Port`